src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-message.service.ts

```diff
@@ -318,6 +318,12 @@ export class TaskMessageService {
       this.rbd_mirroring.pool_peer,
       () => ({})
     ),
+    // RGW operations
+    'rgw/bucket/delete': this.newTaskMessage(this.commonOperations.delete, (metadata) => {
+      return $localize`${
+        metadata.bucket_names.length > 1 ? 'all buckets' : metadata.bucket_names[0]
```

Comment:
If more than one is deleted, that doesn't mean all buckets are deleted. We can select 10 buckets of 100 to be deleted. 

Probably we need to iter over length(if 100 we just need to iter over maybe 10 and the put `...` ) and see how much text we can accommodate in notification

Reply:
Missed this one, but lets keep it generic here if possible

Reply:
This message is on the context of the selected buckets, so `all buckets` refering to all the selected buckets.

If the wording is confusing we can change it.
> Probably we need to iter over length(if 100 we just need to iter over maybe 10 and the put ... ) and see how much text we can accommodate in notification

I don't see a reason to show the name of a few only. So I rather just display a generic message that everything has been deleted when bulk deleting 

@cloudbehl @nizamial09

Reply:
Then probably it would be good to mention all the selected buckets will be deleted.

Reply:
This is a notification that the buckets have been deleted not, the confirmation mofdal already takes care of that

Reply:
'all selected buckets' -> just to be precise? wdyt?

Reply:
changed it to selected buckets